src/RedisSimpleLock.php

```diff
@@ -33,7 +33,10 @@ public function __construct($identifier, Client $client, $ttl = 10000, LoggerInt
         $this->logger     = $logger ?: new NullLogger;
         $this->id         = mt_rand();
         register_shutdown_function($closure = $this->releaseClosure());
-        pcntl_signal(SIGINT, $closure);
+
+        if (php_sapi_name() === 'cli') {
```

Comment:
as said in #6, I'd like that to be opt-in with a constructor argument and let the default be to use `pcntl_signal`